libs/common/src/constants/facility-type.ts

```diff
@@ -8,4 +8,15 @@ export const BSS_EWCS_FACILITY_TYPE = {
   LOAN: 'Loan',
 } as const;
 
+export const BOND_FACILITY_TYPE = {
```

Comment:
Is this not just `bondType`?

Reply:
Good point - just used the existing name, have renamed to `BOND_TYPE`.